src/lib/resource/sections/context-pregnancy.js

```diff
@@ -18,9 +18,9 @@ module.exports = {
+                                '<p class="govuk-body">We need to ask questions about pregnancy related to the crime. These questions may be difficult to answer or may not be relevant to you.</p><p class="govuk-body">By answering them, it ensures you are considered for all correct compensation from us. This helps us better understand the case and if we need to ask for more evidence from you.</p><h2 class="govuk-heading-m">Help and support</h2><p class="govuk-body">You can get practical and emotional support after a crime. Visit <a class="govuk-link" href="https://www.victimandwitnessinformation.org.uk/" target="_blank">Victim and Witness information (opens in new window)</a> if you’re in England and Wales. Or <a class="govuk-link" href="https://www.mygov.scot/victim-witness-support/" target="_blank">support as a victim or witness of crime (opens in new window)</a> in Scotland.</p>{% set templateHtml %}{% include \'contact.njk\' %}{% endset %}{{ govukDetails({summaryText: "If you need help or support with the application",html: \'<p class="govuk-body">You can contact us for help with your application.</p>\' + templateHtml + \'<p class="govuk-body">You can <a class="govuk-link" href="https://www.victimandwitnessinformation.org.uk/" target="_blank">get practical or emotional support (opens in new window)</a> after a crime.</p><p class="govuk-body">There is different practical or emotional support <a class="govuk-link" href="https://www.mygov.scot/victim-witness-support/" target="_blank">if you live in Scotland (opens in new window)</a>.</p>\'})}}',
```

Comment:
Should this say new tab instead of new window?

Reply:
The ticket has been amended the "tabs" is correct